README.md

```diff
@@ -8,3 +8,7 @@ TYPO3 extension to simplify work with Kitodo.Presentation at DDB-Zeitungsportal
 3. Run Grunt: npx grunt
 
 The result is located at `Resources/Public/JavaScript/DdbKitodoZeitungsportal.js
+
+## Copy local configuration from LocalConfig.php
+
+Example of LocalConfig.php is placed in this repository. It can be used as a template during the installation.
```

Comment:
```suggestion
Example of LocalConfiguration.php is placed in this repository. It can be used as a template during the installation.
```

But in fact this doesn't help during the installation. Nobody will copy parts out of the LocalConfiguration.php file. This file is written by TYPO3. It will stay a manual step, adjusting the settings documented under `Documentation/` (that's the place it should go.

All these settings could be done via TypoScript, too. But this has to be changed in Kitodo.Presentation and must be discussed before.